run.sh

```diff
@@ -170,17 +175,39 @@ workload_1_golang() {
                        -api "${API}"
 }
 
+workload_1_nodejs() {
+  . $NVM_HOME/nvm.sh
+  node /usr/bin/nodejs_benchmark_cli/build/internal-tooling/performanceTest.js --project "${PROJECT}" \
+                                                                                     --bucket "${BUCKET_NAME}" \
+                                                                                     --test_type "w1r3" \
+                                                                                     --object_size "${OBJECT_SIZE}..${OBJECT_SIZE}" \
+                                                                                     --workers "${WORKERS}" \
+                                                                                     --output_type "cloud-monitoring" \
+                                                                                     --samples "${SAMPLES}"
+}
+
 workload_2_nodejs() {
   . $NVM_HOME/nvm.sh
-  node /usr/bin/nodejs_benchmark_cli/build/internal-tooling/performanceTest.js --projectid "${PROJECT}" \
+  node /usr/bin/nodejs_benchmark_cli/build/internal-tooling/performanceTest.js --project "${PROJECT}" \
+                                                                                     --bucket "${BUCKET_NAME}" \
+                                                                                     --test_type "range-read" \
+                                                                                     --object_size "${OBJECT_SIZE}..${OBJECT_SIZE}" \
+                                                                                     --range_read_size "${RANGE_READ_SIZE}" \
+                                                                                     --workers "${WORKERS}" \
+                                                                                     --output_type "cloud-monitoring" \
+                                                                                     --samples "${SAMPLES}"
+}
+
+tm_chunk_download_nodejs() {
```

Comment:
Do you think we should define this as an additional workload?

Reply:
I definitely think so. We put some ideas together in the Transfer Manager design docs but I think this probably also needs to get formalized in Frank's doc?